tests/conftest.py

```diff
@@ -205,49 +207,115 @@ def moti_sawr7110_xml():
 <cmml xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="..\Schema\CMML.xsd" version="2.01">
   <head>
     <product operational-mode="official">
-      <title>Observation from BC Meteorological Stations
-      </title>
-      <field>meteorological
-      </field>
-      <category>observation
-      </category>
-      <creation-date refresh-frequency="PT1H">2013-11-29T13:22:57-08:00
-      </creation-date>
+      <title>Observation from BC Meteorological Stations</title>
```

Comment:
Thanks for the formatting. I'm guessing that we just copy/pasted this from a download originally, but this is far easier to read.